lib/menu.rb

```diff
@@ -0,0 +1,21 @@
+class Menu
+
+	attr_accessor :dishes
+
+	def initialize
+		@dishes = []
+	end
+
+	def list_count
+		@dishes.count
+	end
+
+	def add_dish(dish)
+		@dishes << dish
+	end
+
+	def select_dish(dish)
+		@dishes.select{|listed_dish| listed_dish == dish}.first
+	end
+
+end
```

Comment:
this select_dish method seems redundant.  If you already have a reference to the object you already have a reference to the object.  You might be checking that a dish is on the menu, but that is a different thing.  The way you have this here it will throw an error when the dish is not on the menu - not quite what you want - or at least the type of error will be difficult to understand